Source/WTF/wtf/SingleThreadIntegralWrapper.h

```diff
@@ -40,7 +40,10 @@ class SingleThreadIntegralWrapper {
     SingleThreadIntegralWrapper& operator++();
     SingleThreadIntegralWrapper& operator--();
 
+// https://bugs.webkit.org/show_bug.cgi?id=280710
+IGNORE_GCC_WARNINGS_BEGIN("uninitialized")
```

Comment:
How about adding `{ }` to the below code?
> IntegralType m_value { };
It's uninitialzied for the default ctor.

Reply:
Doesn't work. That would have been one of the first things I tried.

Reply:
Yes.